code/style.css

```diff
@@ -1,7 +1,339 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
```

Comment:
I would probably put a min width on the project for it not to get any smaller in width then a small mobile phone. On a iphone 5 your content gets pushed out of the way. so the video is getting "smaller" then the viewwidth.